mmv1/products/compute/Subnetwork.yaml

```diff
@@ -408,3 +408,11 @@ properties:
       existing resources are dropped and prevented from leaving the VPC.
       Setting this field to true will allow these packets to match dynamic routes injected
       via BGP even if their destinations match existing subnet ranges.
+  - !ruby/object:Api::Type::Boolean
```

Comment:
Hi, I see the title as "add support for compute.googleapis.com/Subnetwork to TGC", but this seems to only add one field to the resource which may already exist for TGC?